.github/workflows/ci.yml

```diff
@@ -211,7 +211,7 @@ jobs:
         fi
 
     - name: Upload Coverage Report to Coveralls
-      if: matrix.type.name == 'Debug' && github.repository == 'SFML/SFML' # Disable upload in forks
```

Comment:
This is the only part of this PR that annoys me. Feels a bit ugly to add even more conditions to CI but I don't see a better alternative.